examples/periodic-table/periodic-table.py

```diff
@@ -0,0 +1,133 @@
+from emmo import World
+import ase
+import types
+import owlready2
+import os
+
+
+# define function that returns a string in English in owlready2
+def en(s):
+    """Returns `s` as an English location string."""
+    return owlready2.locstr(s, lang='en')
+
+
+# Load emmo
+world = World()  # to make it extensible for when an ontology gets very large
+emmo = world.get_ontology(
+        'https://raw.githubusercontent.com/emmo-repo/EMMO/periodic-table/'
```

Comment:
This url is not resolved when opening the generated ontology in Protege. We should probably use 
https://emmo-repo.github.io/versions/1.0.0-beta/emmo.ttl instead.